rxbindings/src/main/java/com/amplifyframework/rx/RxStorageBinding.java

```diff
@@ -165,6 +167,16 @@ public Single<StorageListResult> list(@NonNull String path, @NonNull StorageList
         });
     }
 
+    @NonNull
+    @Override
+    public Single<StorageTransferOperation<?, ? extends StorageTransferResult>> getTransfer(
+        @NonNull String transferId) {
+        return toSingle(((onResult, onError) -> {
+            storage.getTransfer(transferId, onResult, onError);
+            return new NoOpCancelable(); // StorageListOperation is not Cancelable at the moment!
```

Comment:
Comment is about StorageListOperation. Should this be returning RxProgressAwareSingleOperation.